pkg/configbuilder/buildconfigfile.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package configbuilder
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/klog"
+	"k8s.io/kops/util/pkg/reflectutils"
+)
+
+// ClientConnectionConfig is used by kube-scheduler to talk to the api server
+type ClientConnectionConfig struct {
+	Burst      *int32   `yaml:"burst,omitempty"`
+	Kubeconfig *string  `yaml:"kubeconfig"`
```

Comment:
It looks like this field has changed names since Kubernetes 1.9, which is the oldest version that kops master branch supports. So it is likely that you'll have to do a Kubernetes version check of some sort, either using the config file only on new versions or changing the format of the file based on the Kubernetes version.

Reply:
Yup, I forgot, thanks!